.appveyor.yml

```diff
@@ -0,0 +1,160 @@
+  - $(APPVEYOR_BUILD_FOLDER)\third_party
+  # Don't compile everything from scratch, it takes too long.
+  - $(APPVEYOR_BUILD_FOLDER)\.mtime_cache
+  - $(DENO_BUILD_PATH)
+  # TODO: Is there a way we can cache these?
+  # - $(APPVEYOR_BUILD_FOLDER)\third_party\v8\third_party\llvm-build
+  # - $(APPVEYOR_BUILD_FOLDER)\third_party\v8\buildtools\win
+
+init:
+  - git config --global core.symlinks true
+
+install:
+  # `Exec` runs a regular executable. It looks at the process' exit code,
+  # rather than its stderr output, to tell if a command has failed
+  - ps: |
+      function Exec([ScriptBlock] $Command) {
+        "$Command".TrimStart(" &") | Write-Host  # Echo command.
+        & $Command 2>&1 | Write-Host             # Execute command.
+        if ($LastExitCode -ne 0) { throw "Failure. Exit code: $LastExitCode" }
+      }
+
+  # `Remove-Cache-Symlinks` removes symlinks from cached directories.
+  # AppVeyor follows symlinks when saving the cache, which breaks things.
+  - ps: |
+      function Remove-Cache-Symlinks {
+        Remove-Item -Path "$env:DENO_BUILD_PATH\node_modules" `
+                    -Recurse -Force -ErrorAction Ignore
+      }
+
+  # Clone git submodules.
+  - git submodule update --init --force --depth 1
+
+  # Prune and pack git objects. Thus when we upload `.git/objects/pack/*` to
+  # the Appveyor cache, it'll include only objects that were actually needed.
+  - git gc
+
+  # Install a recent Node.js version.
+  - ps: Install-Product node 10 x64
+
+  # Make sure the Python version is in PATH, and others are not.
+  - ps: |
+      # Remove the wrong Python version(s) from PATH.
+      $p = $env:PATH -split ";" | Where-Object {
+             -not (Test-Path -Path "$_\python.exe") -and
+             -not (Test-Path -Path "$_\pip.exe")
+           }
+      # Add binary dir for `pip --user` packages.
+      $p += "$env:APPDATA\Python\Scripts"
+      # Add python27-x64.
+      $p += "c:\Python27-x64"
+      $p += "c:\Python27-x64\Scripts"
+      $env:PATH = $p -join ";"
+
+  # Pip on Appveyor is too old. Install a recent version in our user dir.
+  - python -m pip install --upgrade --user pip
+
+  # Install Python packages.
+  - pip install --upgrade --user pywin32 yapf
+
+  # Add Rust/Cargo to PATH.
+  - ps: $env:PATH += ";$env:CARGO_HOME\bin"
+
+  # Install Rust via rustup-init, unless it was restored from cache.
+  - ps: |
+      if (-not (Test-Path -Path $env:CARGO_HOME)) {
+        Invoke-WebRequest -Uri "https://win.rustup.rs" `
+                          -OutFile "$env:TEMP\rustup-init.exe"
+        Exec { & "$env:TEMP\rustup-init.exe" -y }
+      }
+
+  # Update Rust.
+  - rustup update
+
+  # The Rustup directory is very big with many files, which makes saving and
+  # restoring the Appveyor cache noticably slow. Remove unnecessary stuff.
+  # TODO: Use `rustup component remove docs` instead, when this issue
+  # is resolved: https://github.com/rust-lang-nursery/rustup.rs/issues/998.
+  - ps: |
```

Comment:
Link to this issue too https://github.com/ry/deno/issues/386

Reply:
Done (few lines up)